osu.Game/Screens/Play/Player.cs

```diff
@@ -612,6 +612,8 @@ protected void PerformExit(bool showDialogFirst)
             // if an exit has been requested, cancel any pending completion (the user has shown intention to exit).
             resultsDisplayDelegate?.Cancel();
 
+            failAnimationLayer.StopSample();
```

Comment:
While this will work, I feel like it should be combined with the `RemoveFilters` call into a new `public` method (with `RemoveFilters` being changed to `private` and called by the new method).